ci-operator/step-registry/hypershift/mce/heterogeneous/power/destroy/hypershift-mce-heterogeneous-power-destroy-chain.yaml

```diff
@@ -0,0 +1,6 @@
+chain:
```

Comment:
this chain has only one step, so I think we currently don't need to define this chain.

Reply:
removed it, thanks!